cate/core/opimpl.py

```diff
@@ -620,6 +620,10 @@ def _get_geo_spatial_cf_attrs_from_var(ds: xr.Dataset, var_name: str, allow_poin
         return None
 
     var = ds[var_name]
+    res_name = 'geospatial_{}_resolution'.format(var_name)
+    min_name = 'geospatial_{}_min'.format(var_name)
+    max_name = 'geospatial_{}_max'.format(var_name)
+    units_name = 'geospatial_{}_units'.format(var_name)
```

Comment:
For SST CCI, these attribute may also be named differently:

![image](https://user-images.githubusercontent.com/206773/94160312-cad02c00-fe84-11ea-8ccc-afda7ee861f8.png)

Reply:
The 'bbox' attribute names are the names as they appear as datasource metadata. Within the xarray datasets, the terms using 'geospatial' are used.